package.json

```diff
@@ -13,6 +13,7 @@
     "pack-linux": "npm run build && node ./build/script/package.js linux",
     "pack": "npm run pack-win && npm run pack-mac && npm run pack-linux",
     "prepublish": "npm run build && npm run test",
+    "postpublish": "git push && git push --tags",
```

Comment:
A more opinionated command than `git push --tags` would be `git push --follow-tags`. 

From the [docs](https://git-scm.com/docs/git-push):

> Push all the refs that would be pushed without this option, and also push annotated tags in `refs/tags` that are missing from the remote but are pointing at commit-ish that are reachable from the refs being pushed.